File: app/src/main/java/com/minerdev/greformanager/ImageWrapper.java
package com.minerdev.greformanager;

public class ImageWrapper {
    private final Image data;

    public ImageWrapper(Image data) {
        this.data = data;
    }

    public int getId() {
        return data.id;
    }

    public String getCreatedAt() {
        return data.created_at;
    }

    public String getUpdatedAt() {
        return data.updated_at;
    }

    public String getTitle() {
        return data.title;
    }

    public String getPath() {
        return data.path;
    }

    public byte getPosition() {
        return data.position;
    }

    public int getHouseId() {
        return data.house_id;
    }

    public Image getData() {
        return data;
    }
}

File: app/src/main/java/com/minerdev/greformanager/ImageWrapper.kt
package com.minerdev.greformanager

import android.net.Uri

class ImageWrapper(val data: Image) {
    val id: Int
        get() = data.id

    val createdAt: String
        get() = data.createdAt

    val updatedAt: String
        get() = data.updatedAt

    val title: String?
        get() = data.title

    val path: String?
        get() = data.path

    val position: Byte
        get() = data.position

    val thumbnail: Byte
        get() = data.thumbnail

    val houseId: Int
        get() = data.house_id

    fun getUri(position: Int): Uri {
        return Uri.parse(Constants.instance.DNS + "/storage/images/" + data.house_id + "/" + data.title)
    }

}